### PR TITLE
Markup changes to markdown

### PR DESCRIPTION
**Pull request by AurynBeorn**

### CODE REVIEW

File: pages/avsitter2/avsitter2_faces.md [hunk 1]
```diff
@@ -13,27 +13,17 @@ The [AV]faces script allows creators to add facial expressions to AVsitter&trade
 {% include important.html content="Facial expressions can be switched off via the [FACES] button in the [ADJUST] menu." %}
 
 ### Adding Facial Expressions
-To add facial expressions to your furniture, start with the <a href="/avsitter2_home.html#setup">normal setup</a> procedure, then:
-
-<ol>
```
Comment (Sei-Lisa) [inline pages/avsitter2/avsitter2_faces.md:18]: This is an ordered list...

File: pages/avsitter2/avsitter2_faces.md [hunk 2]
```diff
@@ -13,27 +13,17 @@ The [AV]faces script allows creators to add facial expressions to AVsitter&trade
...
-
-<li/>Repeat for each pose you want to give a facial expression.
-
-<li/>When you have finished adding all your facial expressions, click [DUMP] to output your settings into chat.
-
-<li/>Copy-paste the [DUMP] result into your AVpos notecard, replacing the contents of the notecard.
-</ol>
+To add facial expressions to your furniture, start with the [normal setup](/avsitter2_home.html#setup) procedure, then:
+
+- Ensure the [AV]helper and [AV]adjuster are in the prim.
```
Comment (Sei-Lisa) [inline pages/avsitter2/avsitter2_faces.md]: ... but this one is unordered.

**Review by Sei-Lisa — CHANGES_REQUESTED** [review]
Looks good to me, but what about the list type? Was it intentional, or an overlook?

File: pages/avsitter2/avsitter2_faces.md (hunk 2 above)
Comment (AurynBeorn) [reply pages/avsitter2/avsitter2_faces.md]: It was an overlook, sorry! I'll fix it.

**Review by Sei-Lisa — APPROVED** [review]

**Pull request merged**